hooks/cascading-scans/package.json

```diff
@@ -37,15 +37,15 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@kubernetes/client-node": "^0.14.3",
+    "@kubernetes/client-node": "^0.15.0",
     "lodash": "^4.17.21",
     "matcher": "^4.0.0",
     "mustache": "^4.2.0"
   },
   "devDependencies": {
-    "@types/lodash": "^4.14.169",
-    "@types/node": "^14.0.14",
-    "jest": "^27.0.3",
-    "typescript": "^3.9.5"
+    "@types/lodash": "^4.14.171",
+    "@types/node": "^16.3.3",
```

Comment:
We are using node LTS and thus the types should be on latest 14.X

Reply:
Good Catch